src/modules/Shows.js

```diff
@@ -1,91 +1,113 @@
 class Shows {
-  static apiEndPoint = 'https://api.tvmaze.com/shows/1/episodes';
+    static apiEndPoint = 'https://api.tvmaze.com/shows/1/episodes';
 
-  static getShows = async () => {
-    const response = await fetch(this.apiEndPoint).then((response) => response.json());
-    return response;
-  };
+    static getShows = async () => {
+      const response = await fetch(this.apiEndPoint).then((response) => response.json());
+      return response;
+    };
 
-  static updateUI = async () => {
-    const shows = await Shows.getShows();
-    const card = document.getElementById('card');
-    card.classList.add('wrap', 'flex');
-    const commentBody = document.getElementById('comment-section');
+    static getLikes = async () => {
+      const action = {
+        method: 'GET',
+        headers: {
+          'Content-type': 'application/json; charset=UTF-8',
+        },
+      };
+      const response = await fetch(
+        'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/tV364kOhzeIf5RoUn6sV/likes',
```

Comment:
Please consider making changes as requested on the previous pull request. Just declare this URL in a separate variable. ♻️